doc/basic.rst

```diff
@@ -1,14 +1,12 @@
 Basic and Utility Algorithms
 ============================
 
-.. module:: lenskit.algorithms.basic
-
-The :py:mod:`lenskit.algorithms.basic` module contains baseline and utility algorithms
+The :py:mod:`lenskit.algorithms.basic` and :py:mod:`lenskit.algorithms.bias` modules contain baseline and utility algorithms
```

Comment:
I think we should move the `Bias` docs to their own page, `bias.rst`, that should probably go after the `basic.rst` page in the algorithm documentation list.

Reply:
Is this the desired text and format for bias.rst?
![image](https://user-images.githubusercontent.com/7413148/92183386-77316c00-ee0b-11ea-8717-dcb3bee653be.png)

Reply:
Yep!

Reply:
We want the lead paragraph to talk about it just providing the bias model, rather than general baseline and utility algorithms, but yes, it's correct.

We can also delete the Personalized Mean... header.